### ISSUE TITLE
not an issue

### ISSUE THREAD

**Issue by mindforger** [nopeAnon/Automated-HoloCure-Fishing#29]
but kudos to @nopeAnon , @Hexus-One and everyone else contributing ... it works very well now
would have congratulated in any other way if possible :) 

might wanna share how one could contact you for learning some more about what you have done there?

for my part pyautogui would be enough but the performance increase is great but i have a little trouble following the code in it's current state

**Comment by Hexus-One** [comment]
Thanks :) I hope you enjoyed working on this as much as I did! 

> might wanna share how one could contact you for learning some more about what you have done there?

[I livestreamed myself coding the whole thing and talking through the process](https://www.twitch.tv/videos/1906204436) - it's 4.5 hours long with a long of tangents so it's not the best way to efficiently learn 😅 but when I started off learning OpenCV it was a matter of reading through the [tutorials](https://docs.opencv.org/4.x/d6/d00/tutorial_py_root.html) to see what techniques are available to use. Most other things (like learning screen capture and learning how to send input) I learned by googling and researching extensively to see what options there are.
